--- t1aa.etx	2013-08-04 23:14:24.020777939 +0100
+++ t1aa8.etx	2013-08-04 23:29:17.524978715 +0100
@@ -415,15 +415,15 @@
 %       like in a monowidth font.}
 % \endsetslot
 \setslot{longdbls}
- \ligature{LIG}{visiblespace}{germandbls}
- \ligature{LIG/}{comma}{germandbls}
- \ligature{LIG/}{period}{germandbls}
- \ligature{LIG/}{colon}{germandbls}
- \ligature{LIG/}{semicolon}{germandbls}
- \ligature{LIG/}{question}{germandbls}
- \ligature{LIG/}{exclam}{germandbls}
- \ligature{LIG/}{parenright}{germandbls}
- \ligature{LIG/}{quoteright}{germandbls}
+ \ligature{LIG}{visiblespace}{longss}
+ \ligature{LIG/}{comma}{longss}
+ \ligature{LIG/}{period}{longss}
+ \ligature{LIG/}{colon}{longss}
+ \ligature{LIG/}{semicolon}{longss}
+ \ligature{LIG/}{question}{longss}
+ \ligature{LIG/}{exclam}{longss}
+ \ligature{LIG/}{parenright}{longss}
+ \ligature{LIG/}{quoteright}{longss}
    \comment{The ligature `long s' `long s'.}
 \endsetslot
 
@@ -864,7 +864,7 @@
 % \Fi
 \setslot{\lc{S}{longs}}
    \ligature{LIG}{visiblespace}{\lc{S}{s}}
-   \ligature{LIG}{\lc{S}{s}}{germandbls}
+   \ligature{LIG}{\lc{S}{s}}{longss}
    \ligature{LIG}{\lc{S}{longs}}{longdbls}
    \ligature{LIG}{\lc{H}{h}}{longsh}
    \ligature{LIG}{\lc{I}{i}}{longsi}
@@ -879,6 +879,8 @@
    \ligature{LIG/}{parenright}{\lc{S}{s}}
    \ligature{LIG/}{quoteright}{\lc{S}{s}}
    \ligature{LIG/}{\lc{F}{f}}{\lc{S}{s}}
+   \ligature{LIG/}{\lc{B}{b}}{\lc{S}{s}}
+   \ligature{LIG/}{\lc{K}{k}}{\lc{S}{s}}
    \comment{The letter `long s'.}
 \endsetslot
 
@@ -1471,8 +1473,11 @@
       in \plain\ \TeX.}
 \endsetslot
 
-\setslot{\lc{SS}{germandbls}}
-   \comment{The letter `\ss'.}
+% \setslot{\lc{SS}{germandbls}}
+%    \comment{The letter `\ss'.}
+% \endsetslot
+\setslot{\lclig{SS}{longss}}
+   \comment{The ligature `long s' `{s}'.}
 \endsetslot
 
 \endencoding
